assets/App.vue

```diff
@@ -168,6 +174,9 @@ export default {
   position: fixed;
   left: 10px;
   bottom: 10px;
+  & svg {
```

Comment:
Do you need this if use class `.icon` from bulma? I think it does it for you.

Reply:
You're right looks like that will do it. Note that it's going to be a bit smaller now.

<img width="319" alt="image" src="https://user-images.githubusercontent.com/3276350/140661419-9de2854d-86a2-45d2-b42e-67cf3f243f53.png">